src/Rules/Resources/Descriptions.resx

```diff
@@ -122,12 +122,12 @@
     <comment>The description of an accessibility problem with an element currently under inspection. {Locked="BoundingRectangle"}</comment>
   </data>
   <data name="ButtonWithSplitButtonParentPattern" xml:space="preserve">
-    <value>A button element should only support one of the Invoke, Toggle, or ExpandCollapse patterns when a splitbutton is the parent. </value>
+    <value>A button element should only support one of the Invoke, Toggle, or ExpandCollapse patterns when a split button is the parent. </value>
```

Comment:
`SplitButton` is a UIA-defined ControlType (https://learn.microsoft.com/en-us/windows/win32/winauto/uiauto-supportsplitbuttoncontroltype). The error here is one of casing, not of a missing space